README.md

```diff
@@ -23,7 +23,13 @@ In order to deploy the solutions we will need to activate a config-controller in
 
 ### arete cli Install
 
-See arete [README](./cli/README.md) for installation instructions.
+See arete [README](./cli/README.md) for instructions to build from source.
+
+```
+wget https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/releases/download/v0.0.3-alpha/arete
```

Comment:
verified

Reply:
Ya wanted to keep it basic, couldn't guarantee go would be installed and it's not required. The alternate option would be to add the current dir to $PATH.